packages/kbn-dom-drag-drop/src/constants.ts

```diff
@@ -8,4 +8,4 @@
 
 export const DEFAULT_DATA_TEST_SUBJ = 'domDragDrop';
 export const REORDER_ITEM_HEIGHT = 32;
-export const REORDER_ITEM_MARGIN = 8;
+export const REORDER_ITEM_MARGIN = 16;
```

Comment:
Would it make sense to link these values to euiTheme values? https://elastic.github.io/eui/#/theming/sizing/values

Reply:
With this you should be able to access the same variables:

```
import { euiThemeVars } from '@kbn/ui-theme';
```

Reply:
Ahhh, yes, but the variables here are stored as strings with `px` included (for example 16px) and we make calculations in JS later based on these values. I feel it's not worth to use it here. I'd prefer to leave it as is 🙏🏼